### PR TITLE
fix: do not create empty folders in assets

### PR DESCRIPTION
**Pull request by tikhanovichA**
https://oat-sa.atlassian.net/browse/ADF-1606

This PR will fix creating empty folder for assets. 

How to check:

1. Import Test from preconditions
3. Open Assets tab

Checklist:
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

### CODE REVIEW

**Review by pnal — CHANGES_REQUESTED** [review]
Good job. The fix works, tested on my local environment.

File: test/unit/model/qti/asset/factory/SharedStimulusFactoryTest.php [hunk 1]
```diff
@@ -74,11 +81,23 @@ public function testCreateShardedStimulusFromSourceFiles(): void
...
+            ->expects($this->once())
+            ->method('getClass')
+            ->willReturn($mediaRootClassMock);
+
         $result = $this->subject->createShardedStimulusFromSourceFiles(
             'file.xml',
             'path/to/xml/filename.xml',
             'absolute/path/to/asset/passage.xml',
-            $this->classMock
+            ['item label']
```
[the remark this answers is not in the document]
Comment (tikhanovichA) [reply test/unit/model/qti/asset/factory/SharedStimulusFactoryTest.php]: Added more test cases

File: model/qti/ImportService.php [hunk 2]
```diff
@@ -94,7 +94,7 @@ class ImportService extends ConfigurableService
 
     public const PROPERTY_QTI_ITEM_IDENTIFIER = 'http://www.tao.lu/Ontologies/TAOItem.rdf#QtiItemIdentifier';
 
-    private const MEDIA_ROOT_CLASS = 'http://www.tao.lu/Ontologies/TAOMedia.rdf#Media';
+    public const MEDIA_ROOT_CLASS = 'http://www.tao.lu/Ontologies/TAOMedia.rdf#Media';
```
Comment (pnal) [inline model/qti/ImportService.php:97]: Since you do not use it here anymore, can you move it to `SharedStimulusFactory` and leave private?

**Review by pnal — APPROVED** [review]
Thanks for changes, left last one comment

File: model/qti/ImportService.php (hunk 2 above)
Comment (tikhanovichA) [reply model/qti/ImportService.php:97]: good catch, thanks

**Pull request merged**